wpilibc/src/main/native/include/frc/motorcontrol/PWMMotorController.h

```diff
@@ -71,6 +90,13 @@ class PWMMotorController : public MotorController,
    */
   void EnableDeadbandElimination(bool eliminateDeadband);
 
+  /**
+   * Make the given PWM motor controller follow the output of this one.
+   *
+   * @param follower The motor controller follower.
+   */
+  void AddFollower(PWMMotorController& follower);
```

Comment:
It might be nice if there was also a move overload?  I know that makes the vector more complicated (e.g. we'd probably have to have both owning and non-owning vectors), but it would be nice from an API experience to just say e.g. `AddFollower(Talon(2))` or something?